ui/imports/shared/views/NetworkCardsComponent.qml

```diff
@@ -131,12 +135,12 @@ Item {
                     objectName: model.chainId
                     property bool isOnBestRoute: false
                     property double amountToReceive: 0
+                    property bool preferred: store.preferredChainIds.includes(model.chainId)
                     primaryText: model.chainName
                     secondaryText: LocaleUtils.numberToLocaleString(amountToReceive)
-                    tertiaryText: ""
-                    // To-do preferred in not something that is supported yet
-                    state: root.errorMode ? "error" : "default"
-                    // opacity: preferred ? 1 : 0
+                    tertiaryText: state === "unpreferred"  ? qsTr("unpreferred").toUpperCase() : ""
```

Comment:
`qsTr("unpreferred").toUpperCase()` 

No just no, some languages don't even have uppercase letters :D

Reply:
```suggestion
                    tertiaryText: state === "unpreferred" ? qsTr("UNPREFERRED") : ""
```

Reply:
hahahaha yes! again thanks for finding this!